tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RegionalAuthIntegrationTests.cs

```diff
@@ -103,6 +103,73 @@ public async Task AcquireTokenToRegionalEndpointAsync()
                 AuthenticationResult result = await CreateAuthenticationResultAsync().ConfigureAwait(false); // regional endpoint
                 AssertTokenSource_IsIdP(result);
                 AssertValidHost(true, factory);
+                AssertTelemetry(factory, "2|1004,0|centralus,1,0,");
+            }
+            finally
+            {
+                Environment.SetEnvironmentVariable(TestConstants.RegionName, null);
```

Comment:
Would it be appropriate to put this line into a separate test cleanup method, so it doesn't have to be repeated?

Reply:
Updated